src/components/Utils/getCreateInventorySetting.js

```diff
@@ -0,0 +1,22 @@
+import { get } from 'lodash';
+
+const getCreateInventorySetting = (configs) => {
+  let createInventorySetting = get(configs, [0, 'value'], '{}');
+  const config = {
+    eresource: 'Instance, Holding',
```

Comment:
I think sooner or later we will need these values on UI to provide different behavior. So it might be a good idea to make values as constants

Reply:
Done